src/validations/rules/ssp.sch

```diff
@@ -1661,6 +1608,46 @@
                 test="current() = $security-objective-levels">[Section B Check 3.10] A FedRAMP SSP must specify an allowed security objective
                 value.</sch:assert>
         </sch:rule>
+        <sch:rule
```

Comment:
Why was this rule placed within the `fips-199` pattern? It has nothing to do with FIPS 199. The correct pattern is `control-implementation`.